### DISCUSSION TITLE
Setting to disable the changelog after update

### DISCUSSION THREAD

**Discussion by rlnt** [izhangzhihao/intellij-rainbow-brackets#2621] [Q&A] [answered]
## Expected Behavior
Don't open a window with the changelog after the plugin updates. Or at least add a setting for it.

## Current Behavior
It opens a window with the changelog after updating.

**Comment by izhangzhihao** [comment, marked as answer]
Please follow https://github.com/izhangzhihao/intellij-rainbow-brackets/issues/279#issuecomment-694782959

```xml
<application>
  <component name="RainbowSettings">
    <option name="showNotificationOnUpdate" value="false" />
  </component>
</application>
```

The config path is here: https://github.com/izhangzhihao/intellij-rainbow-brackets#config-file-path
